src/ACF/Image.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+# -*- coding: utf-8 -*-
+
+declare(strict_types=1);
+
+namespace MultisiteGlobalMedia\ACF;
+
+use MultisiteGlobalMedia\Helper;
+use MultisiteGlobalMedia\Site;
+use MultisiteGlobalMedia\SiteSwitcher;
+
+class Image
+{
+    use Helper;
+
+    /**
+     * @var Site
+     */
+    private $site;
+
+    /**
+     * @var SiteSwitcher
+     */
+    private $siteSwitcher;
+
+    /**
+     * @var \ACF_Data
+     */
+    private $store;
+
+    /**
+     * Integration for ACF, specifically Image fields
+     *
+     * @param Site $site
+     * @param SiteSwitcher $siteSwitcher
+     */
+    private function __construct(Site $site, SiteSwitcher $siteSwitcher)
+    {
+        $this->site = $site;
+        $this->siteSwitcher = $siteSwitcher;
+    }
+
+    public static function bootstrap(Site $site, SiteSwitcher $siteSwitcher)
+    {
+        $image = new Image($site, $siteSwitcher);
+
+        // ACF can be included within a theme too - check in after_setup_theme action
+        // https://www.advancedcustomfields.com/resources/including-acf-within-a-plugin-or-theme/
+        \add_action('after_setup_theme', [$image, 'afterSetupTheme']);
+
+        return $image;
+    }
```

Comment:
The bootstrap should be performed by the Plugin, same for `afterSetupTheme`, see https://github.com/bueltge/multisite-global-media/blob/master/src/Plugin.php#L63-L65 as an example.

The class must not perform more than one thing and in any case nothing concerning the intergration with the Application which in this case is WP.